media-libs/boca/boca-1.0~alpha20180913.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="An audio component framework and library"
+DESCRIPTION="BoCA is the component framework behind the fre:ac audio \
+converter. It provides unified interfaces for components like encoders, \
+decoders, taggers and extensions as well as code to support communication \
+between applications and their components."
+HOMEPAGE="https://github.com/enzo1982/BoCA"
+COPYRIGHT="2007-2018 Robert Kausch"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/enzo1982/BoCA/releases/download/v1.1-alpha-20180913/freac-cdk-1.1-alpha-20180913.tar.gz"
+CHECKSUM_SHA256="d8a6f59e3ccc68b3dbe5d77e5f3e97240691196467656714c678ccbf2932257b"
+SOURCE_DIR="freac-cdk-1.1-alpha-20180913"
```

Comment:
Was it on purpose that you named the recipe `boca-1.0~alpha20180913.recipe` ? (I guess the "1.0" was a typo and that you meant "1.1".)
Because if you rename it to `boca-1.1~alpha20180913.recipe` then you can make some nice changes:
```diff
-SOURCE_URI="https://github.com/enzo1982/BoCA/releases/download/v1.1-alpha-20180913/freac-cdk-1.1-alpha-20180913.tar.gz"
+SOURCE_URI="https://github.com/enzo1982/BoCA/releases/download/v${portVersion/\~alpha/-alpha-}/freac-cdk-${portVersion/\~alpha/-alpha-}.tar.gz"
CHECKSUM_SHA256="d8a6f59e3ccc68b3dbe5d77e5f3e97240691196467656714c678ccbf2932257b"
SOURCE_DIR="freac-cdk-${portVersion/\~alpha/-alpha-}"
```

Reply:
Oops, we need another tiny change:
```diff
-libVersion="${portVersion%~*}"
+libVersion="1.0"
```
otherwise the above suggestion will produce a policy error if we run haikuporter with the `--strict-policy` switch or with its short equivalent `-S`.

Reply:
The 1.0 version number is intentional. The library is BoCA v1.0, but it's part of the fre:ac Component Development Kit v1.1.

I will fix this in the next release by offering a separate boca archive.